src/components/nodes/boundaryEvent/index.js

```diff
@@ -44,7 +44,6 @@ export default {
               component: 'FormInput',
               config: {
                 ...nameConfigSettings,
```

Comment:
I wonder if destructuring is still needed in this situation. Could you assign the config directly?
```js
config: nameConfigSettings,
```

This could apply to a few other places as well.

Reply:
Great point, thanks. Updated to do this and all seems good.